fuzzing/tokens/fuzz.go

```diff
@@ -57,7 +57,7 @@ func newToken(tg *handshake.TokenGenerator, data []byte) int {
 		}
 	}
 	start := time.Now()
-	encrypted, err := tg.NewToken(addr)
```

Comment:
You need to reslice `data` here, so that you don't use the same field for IP and RTT.

Reply:
This too.